components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -55,7 +62,8 @@ export class ProjectListComponent implements OnInit, OnDestroy {
       name: ['', Validators.required],
       id: ['',
         [Validators.required, Validators.pattern(Regex.patterns.ID), Validators.maxLength(48)]],
-      addPolicies: [true]
+      addPolicies: [true],
```

Comment:
❓ Does this not imply a default of true? I feel like we've stated that twice (with `addTeams = true` above)

Reply:
Yeah, it's a bit weird. This is the constructor for the FormBuilder which exists outside of the generic add modal component and is passed in. `addTeams` in that component is just a boolean we keep in sync with the checkbox state that we then use to drive the form. The first thing that component does is change these form values to true so defaulting them here does feel pointless.